plugins/riemann-controller/riemann_controller/resources/manager.config

```diff
@@ -77,7 +77,9 @@
 
 (defn- amqp-connection-conf
   []
-  {:host (or (System/getenv "RABBITMQ_HOST") "localhost")})
+  {:host (or (System/getenv "RABBITMQ_HOST") "localhost")
+   :username (or (System/getenv "RABBITMQ_USER") "testuser")
```

Comment:
err.. I don't know why.. but testuser/testpass bothers me. Maybe "cloudify" "cl0ud1fy" or something? To make it less generic... ? @noakup ?